spec/movie_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../classes/movie'
+
+movie = Movie.new('2020/2/2')
+
+describe '#Review the Movie file' do
+  it 'returns true' do
+    expect(movie.silent).to eql false
+  end
+
+  it 'returns true' do
+    expect(movie.can_be_archived?).to eql false
+  end
+end
```

Comment:
Hi @j-tee 

- [ ]  Since the `can_be_archived?` method is a private method, then what you should test instead is the `move_to_archive` method, which uses the `can_be_archived?` and the class you implemented inherit from the `item`.